main.js

```diff
@@ -30,5 +30,29 @@ client.on('ready', () => {
         }
     });
 });
+client.on('message', message => { //Ten event wykonuje się, gdy bot wykryje wiadomość
```

Comment:
te komentarze to chyba są zbędne - czasem warto wrzucić jakiś komentarz, gdy trzeba coś wyjaśnić, ale tu każdy ewidentnie wie co ten kod robi